### PR TITLE
Fix deprecated annotation

### PR DESCRIPTION
**Pull request by itaykl**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -6,19 +6,19 @@ buildscript {
...
+    id 'com.jfrog.bintray' version '1.8.5'
+    id 'net.researchgate.release' version '2.8.1'
+    id 'com.github.hierynomus.license' version '0.15.0'
 }
 
 apply plugin: 'java'
 apply plugin: 'idea'
 
-apply plugin: 'osgi'
+//apply plugin: 'osgi'
```
Comment (DoctorVoid) [inline build.gradle]: comment

File: src/test/java/graphql/annotations/processor/retrievers/fieldBuilders/DeprecateBuilderTest.java [hunk 2]
```diff
@@ -95,4 +103,43 @@ public void build_noDeprecatedAnnotation_returnNull() throws NoSuchMethodExcepti
         // assert
         assertNull(deprecate);
     }
+
```
Comment (DoctorVoid) [inline src/test/java/graphql/annotations/processor/retrievers/fieldBuilders/DeprecateBuilderTest.java:106]: Add test for deprecated on type

**Review by DoctorVoid — CHANGES_REQUESTED** [review]

File: src/test/java/graphql/annotations/processor/retrievers/fieldBuilders/DeprecateBuilderTest.java [hunk 3]
```diff
@@ -43,6 +44,14 @@ public int testMethod4() {
         return 1;
     }
 
+    @GraphQLDeprecate("test field deprecated")
```
Comment (DoctorVoid) [inline src/test/java/graphql/annotations/processor/retrievers/fieldBuilders/DeprecateBuilderTest.java]: make codecov happy by lifting those fields to the top of the file above the resolver functions

**Comment by osher-sade** [comment]
Check if the main README needs to be updated

File: src/test/java/graphql/annotations/processor/retrievers/fieldBuilders/DeprecateBuilderTest.java [hunk 4]
```diff
@@ -95,4 +103,43 @@ public void build_noDeprecatedAnnotation_returnNull() throws NoSuchMethodExcepti
         // assert
         assertNull(deprecate);
     }
+
+    @Test
+    public void build_deprecatedAnnotationExistsOnField_returnNull() throws NoSuchFieldException {
```
Comment (yarinvak) [inline src/test/java/graphql/annotations/processor/retrievers/fieldBuilders/DeprecateBuilderTest.java]: does not exist

**Review by yarinvak — APPROVED** [review]

**Pull request merged**